dataset_pack.py

```diff
@@ -0,0 +1,65 @@
+import argparse
+import yaml
+from pathlib import Path
+import os
+import shutil
+import cv2
+
+def clean(dataset_yaml_in, output_folder):
+    robot_names = [str(f).split('/')[-1] for f in Path(dataset_yaml_in).parent.iterdir() if f.is_dir()]
+    output_folder = Path(output_folder)
+    if output_folder.exists() and output_folder.is_dir():
+        shutil.rmtree(output_folder)
```

Comment:
please don't use that - it's dangerous. Just reporting an error is fine, if the output folder exists.